integration-tests/src/test/java/oracle/weblogic/kubernetes/ItRetryImprovements.java

```diff
@@ -176,12 +188,132 @@ void testRetryStoppedAfterIssueFixed() {
     logger.info("Create secret for admin credentials");
     createSecretWithUsernamePassword(wlSecretName, domainNamespace, ADMIN_USERNAME_DEFAULT, ADMIN_PASSWORD_DEFAULT);
 
+    verifyDomainExistsAndServerStarted(replicaCount);
+  }
+
+  /**
+   * Create a domain-in-image domain with an invalid domain resource that has duplicate server names
+   * the domain should fail to start with SEVERE error and FatalDomainInvalidError
+   * the Operator should stop retrying immediately.
+   * Also log a clear message in Operator logs and the domain Failed condition with the cause,
```

Comment:
Also log a clear message in Operator logs  --> Also a message is logged into Operator Log indicating ...

Reply:
Changed